profile_xf05id1/startup/15-optics.py

```diff
@@ -2,9 +2,24 @@
 from ophyd import Device
 from ophyd import Component as Cpt
 
+class SRXSlitsWB(Device):
+    #THIS IS BROKEN AT EPICS :LEVEL
```

Comment:
As @klauer pointed out in the group meeting yesterday, we should track hardware issues that we are forced to (or decide to) paper over in Python by opening GitHub issues against the beamline's IPython profile repos.